src/components/RegisterForm/index.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Input, Button, Select } from 'antd';
+import { NavLink } from 'react-router-dom';
+
+import './style.less';
+import logo from '../../assets/graphics/logo.svg';
+
+const { Option } = Select;
+
+function getYears() {
+  let currentYear = new Date().getFullYear();
+  let years = [];
+  for (var i = currentYear; i < currentYear + 6; i++) {
+    years.push(i.toString());
+  }
+  return years.concat(['Other']);
```

Comment:
Added the dynamic years

Reply:
Needs to be an int (not a string) within 6 six years of the current year (see schema [here](https://github.com/acmucsd/membership-portal/blob/master/app/db/models/user.js#L106) and example User instances [here](https://github.com/acmucsd/membership-portal/blob/master/app/db/populateDb.js)).

Also, the actual function could be a little tighter, e.g.
```javascript
function getYears() {
    const currentYear = new Date().getFullYear();
    return [...Array(6)].map((_,i) => i + currentYear);
}
```